sql/DatabaseInfo.md

```diff
@@ -0,0 +1,15 @@
+Database consists of one table, `datestoreserve`.
+
+Table creation statement:
+
+```
+CREATE TABLE `datestoreserve` (
+ `month` int(11) NOT NULL,
+ `day` int(11) NOT NULL,
+ `year` int(11) NOT NULL,
+ `mountains` varchar(255) NOT NULL,
+ `emails` varchar(255) NOT NULL
+)
+```
+
+The columns `month`, `day`, and `year` combine to form a specific date that the script should reserve. Values in the columns `mountains` and `emails` filter the script so that it will only reserve a specific date for mountains present in the `mountains` column and emails present in the `email` column. The `mountains` and `emails` columns should be formatted as a comma delimited string. For example, a valid value for `mountains` is "Brighton, Arapahoe Basin".
```

Comment:
Yea ideally, we would do some sort of JOINING or whatever (I also forgot relational db stuff). But this is good for now!

Reply:
Also, what is the purpose of adding the email to the database, since the script will only reserve for the email given to main.py? Are you thinking you could populate the database with emails for different ikon users and run the script again with different logins? Is it to avoid the inconvenience of changing the database for each ikon user?

Reply:
Last thing, can you change the insertDatesToReserve, deleteDatesToReserve and insertWeekends python scripts to take the mountain (and email if we keep that) as arguments and then add them to the database? Also you need to update the setup/sqlSetup.sql file to create this modified table. Maybe get rid of the sqlSetup.sql file and just add the stuff in there to this DatabaseInfo.md. Then you need to update the readme file to say "run the sql commands in sql/DatabaseInfo.md" instead of (under linux install instructions) "run these commands in mysql" (but keep the 'in my sql part) and (under Windows install instructions) "run the sql commands in setup/sqlSetup.sql"

Reply:
Ok glad I'm not the only one lol. And yeah you got it right exactly cuz right now I'm running the script for myself Morgan and Leslie at the same time and Leslie said she's not interested in certain dates that Morgan and I are interested in so that's why I added that feature. And figured at the same time might as well add a mountains column too.

Reply:
I actually completely missed that there was a sqlSetup.sql file haha. I just updated the creation statement there. Also I just removed the explanatory comment that I had in DatabaseInfo.md for now. I figured there isn't any preexisting documentation explaining the implementation of the script right now and that's what that comment was so devs can just read the code and figure it out.